.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,5 +4,6 @@
 
 - [ ] Changes are clearly described above
 - [ ] Any relevant issue threads are referenced in the description
-- [ ] Any new features are tested (see [docs/DEVEL.md](docs/DEVEL.md) for details)
+- [ ] Any new features are tested (see the [development manual](https://microhapulator.readthedocs.io/en/stable/devel.html) for details)
+- [ ] CLI documentation (see [docs/cli.md](docs/cli.md)) and Python API documentation (see [microhapulator/api.py](microhapulator/api.py)) are up-to-date and in sync
```

Comment:
Also updated the PR template because I keep on *almost* forgetting to update API documentation. It's only a matter of time...